app/javascript/packages/document-capture/components/review-issues-step.jsx

```diff
@@ -142,6 +166,39 @@ function ReviewIssuesStep({
       <DesktopDocumentDisclosure />
       <StartOverOrCancel />
     </>
+  ) : (
+    <Warning
+      heading={t('errors.doc_auth.throttled_heading')}
+      actionText={t('idv.failure.button.warning')}
+      actionOnClick={onWarningPageDismissed}
+      troubleshootingOptions={
+        /** @type {TroubleshootingOption[]} */ ([
+          {
+            url: documentCaptureTipsURL,
+            text: t('idv.troubleshooting.options.doc_capture_tips'),
+            isExternal: true,
+          },
+          spName && {
+            url: getFailureToProofURL('post_submission_warning'),
+            text: t('idv.troubleshooting.options.get_help_at_sp', { sp_name: spName }),
+            isExternal: true,
+          },
+        ].filter(Boolean))
+      }
+    >
+      {!!unknownFieldErrors &&
+        unknownFieldErrors.map(({ error }) => <p key={error.message}>{error.message}</p>)}
+
+      {remainingAttempts <= 3 && (
```

Comment:
Minor: Might be good idea to pull out this magic number as a constant toward the top of the file.

Reply:
Yeah, good call. I thought about making it a config but then decided against it and just didn't think about pulling it out as a constant.